modules/basis_universal/register_types.cpp

```diff
@@ -30,6 +30,7 @@
 
 #include "register_types.h"
 
+#include "core/config/project_settings.h"
 #include "image_compress_basisu.h"
```

Comment:
In modules we keep the local module includes separate from the "absolute" common Godot headers.
```suggestion
#include "image_compress_basisu.h"

#include "core/config/project_settings.h"
```